test/foundry-tests/Tick.t.sol

```diff
@@ -105,18 +105,18 @@ contract TickTest is Test, GasSnapshot {
         checkCantOverflow(HIGH_TICK_SPACING, maxLiquidityPerTick);
     }
 
-    function testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueFor1() public {
-        uint128 maxLiquidityPerTick = tickSpacingToMaxLiquidityPerTick(1);
+    function testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForMinTickSpacing() public {
+        uint128 maxLiquidityPerTick = tickSpacingToMaxLiquidityPerTick(Constants.MIN_TICK_SPACING);
 
         assertEq(maxLiquidityPerTick, 191757530477355301479181766273477); // 126 bits
-        checkCantOverflow(1, maxLiquidityPerTick);
+        checkCantOverflow(Constants.MIN_TICK_SPACING, maxLiquidityPerTick);
     }
 
-    function testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForEntireRange() public {
-        uint128 maxLiquidityPerTick = tickSpacingToMaxLiquidityPerTick(887272);
+    function testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForMaxTickSpacing() public {
+        uint128 maxLiquidityPerTick = tickSpacingToMaxLiquidityPerTick(Constants.MAX_TICK_SPACING);
 
-        assertEq(maxLiquidityPerTick, Constants.MAX_UINT128 / 3); // 126 bits
```

Comment:
Do you mind adding the additional test you mention in #371 to this test file as well? 
```
function testTick_tickSpacingToMaxLiquidityPerTick_returnsTheCorrectValueForEntireRange() public {
    uint128 maxLiquidityPerTick = tickSpacingToMaxLiquidityPerTick(Constants.MAX_TICK);

    assertEq(maxLiquidityPerTick, type(uint128).max / 3);
    checkCantOverflow(Constants.MAX_TICK, maxLiquidityPerTick);
}
```